app/managers.py

```diff
@@ -0,0 +1,36 @@
+import sqlite3
+
+from app.models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.db3")
+
```

Comment:
You haven't fixed the previous comment regarding the table name.

Create an attribute to store the table name  and use it throughout your code.


```suggestion
self.table_name = "actors"
```